apps/qubit/modules/settings/actions/privacyNotificationAction.class.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ class SettingsPrivacyNotificationAction extends DefaultEditAction
+ {
+   // Arrays not allowed in class constants
+   public static
+     $NAMES = array(
+       'privacy_notification_enabled',
+       'privacy_notification');
+
+   protected function earlyExecute()
+   {
+     $this->i18n = sfContext::getInstance()->i18n;
+   }
+
+   protected function addField($name)
+   {
+     switch ($name)
+     {
+       case 'privacy_notification':
+         // Determine default value
+         $default = (null !== $this->$name = QubitSetting::getByName($name))
+           ? $this->$name->getValue()
+           : '';
+           
+         $this->form->setDefault($name, $default);
+         $this->form->setValidator($name, new sfValidatorString(array('required' => true)));
+         $this->form->setWidget($name, new sfWidgetFormInput);
+
+         break;
+
+       case 'privacy_notification_enabled':
+         // Determine default value
+         $defaults = array('privacy_notification_enabled' => 1);
+
+         $default = (null !== $this->$name = QubitSetting::getByName($name))
+           ? $this->$name->getValue(array('sourceCulture' => true))
+           : $defaults[$name];
+
+         $options = array($this->i18n->__('No'), $this->i18n->__('Yes'));
```

Comment:
We should normalize this some day, I prefer them this way but we have them a lot as 'yes' and 'no' (lowercase).